src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -214,6 +213,7 @@ const ArtistTitle = styled(InvertedSerif)`
 `
 
 const ReadMoreButton = styled(InvertedSans)<{ onClick: () => void }>`
+  text-decoration: underline;
```

Comment:
This is sitting much closer to the text than in the image in the ticket, I think we should go with a border-bottom here to give some breathing room.